src/instruments/src/EFB/Navigation/Pages/LocalFilesPage/LocalFileChartSelector.tsx

```diff
@@ -59,26 +63,35 @@ export const LocalFileChartSelector = ({ selectedTab, loading }: LocalFileChartS
         );
     }
 
-    const getChartResourceUrl = async (chart: LocalFileChart) => {
-        const resp = await fetch(chart.type === 'PDF'
-            ? `http://localhost:8380/api/v1/utility/pdf?filename=${chart.fileName}&pagenumber=1`
-            : `http://localhost:8380/api/v1/utility/image?filename=${chart.fileName}`);
-
-        if (!resp.ok) {
+    const getChartResourceUrl = async (chart: LocalFileChart): Promise<string> => {
+        if (simbridgeEnabled !== 'AUTO ON') {
+            return Promise.reject();
+        }
+        try {
+            if (chart.type === 'PDF') {
+                const blob = await Viewer.getPDFPage(chart.fileName, 1);
+                return URL.createObjectURL(blob);
+            }
+            const blob = await Viewer.getImage(chart.fileName, 1);
+            return URL.createObjectURL(blob);
```

Comment:
can we make `getPDFPage` and `GetImage` return the objectURL string rather than a block for further abstraction.

Reply:
You mean change the simbridge-client API?

Reply:
yes

Reply:
done